### PR TITLE
Modular Armour Tweaks

### PR DESCRIPTION
**Pull request by Clem-The-Duck**
Changes the slowdown of modular armour:

- The plate accessory slowdown has been changed from 1 to 0.4
- The leg guard accessory slowdown has been changed from 0 to 0.3
- the arm guard accessory slowdown has been changed from 0 to 0.3

When wearing a full set of armour, your slowdown should be equal to 1 which is the current slowdown, taking off the leg and arm guards will now allow you to move faster at the cost of coverage. This allows better customisation and gives more incentive to play around with the modular side of the modular armour system. Note that this slowdown does not apply to light armour like the standard corporate armour and similar.

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -216,10 +216,10 @@
 	icon_state = "helm_heavy"
 	item_state = "helm_heavy"
 	armor = list(
-		melee = ARMOR_MELEE_VERY_HIGH,
-		bullet = ARMOR_BALLISTIC_RIFLE,
+		melee = ARMOR_MELEE_MAJOR,
+		bullet = ARMOR_BALLISTIC_MAJOR,
 		laser = ARMOR_LASER_MEDIUM,
-		energy = ARMOR_ENERGY_RESISTANT,
+		energy = ARMOR_ENERGY_SMALL,
```
Comment (FluffyGhoster) [inline code/modules/clothing/head/helmet.dm:222]: If you want to change the merc armor, do it in another PR, first to atomize it and second because it goes directly to maintainer discussion since they don't need any less survivability, quite the contrary

Comment (Clem-The-Duck) [reply code/modules/clothing/head/helmet.dm:222]: i dont see why it would need a maintainer discussion since this would bring it inline with the Merc Heavy armour vests current values. Instead of randomly having ballistic and riot armour level protection. With the new values, merc armour is no less stronger than sec heavy armour, just consistent.

Comment (FluffyGhoster) [reply code/modules/clothing/head/helmet.dm:222]: Because I don't agree with it being reduced, as I said they don't need lower survivability, quite the contrary

Comment (Clem-The-Duck) [reply code/modules/clothing/head/helmet.dm:222]: I dont understand why the merc helmet needs such high ballistic and melee stats (equivalent to sec ballistic and riot armour) while the plate carrier uses the standard for heavy armour stats. Its an inconsistency among all our current heavy armours, this is just correcting it. Practically this doesent change their overall survivability too, since their heavy armour plates are unchanged. Again this is for the sake of consistency with our armour sets, i still do not see the issue.

Comment (FluffyGhoster) [reply code/modules/clothing/head/helmet.dm:222]: We can buff the plate carrier if need be, they however don't have an easy way to heal their brain (no akysine or much of medical support) so it makes sense to protect it more. Either way, this needs to be splitted in a different PR if you want to propose it.

Comment (Clem-The-Duck) [reply code/modules/clothing/head/helmet.dm:222]: i can split it into another PR, but i disagree on buffing the plate carrier. Mercs have Sanna to heal and high DPS weapons to keep security at bay. Buffing merc armour and only merc armour also goes agains the consistency in armour types we currently have, with light - > heavy - > specialist type armour.

File: code/modules/clothing/under/accessories/armor.dm [hunk 2]
```diff
@@ -94,11 +98,13 @@
 		energy = ARMOR_ENERGY_SMALL,
 		bomb = ARMOR_BOMB_PADDED,
 	)
+	slowdown = 0.3
 
 /obj/item/clothing/accessory/leg_guard/heavy/scc
 	name = "heavy SCC leg guards"
 	icon_state = "legguards_blue"
 	item_state = "legguards_blue"
+	slowdown = 0
```
Comment (FluffyGhoster) [inline code/modules/clothing/under/accessories/armor.dm:107]: Supposed to be 0?

Comment (Clem-The-Duck) [reply code/modules/clothing/under/accessories/armor.dm:107]: i kept the 0 slowdown on all the heavy SCC armour pieces as right now they are admin spawn only and have only been used in maybe one or two events. Though I'll probably change them all to match the other sets incase they see usage again.

**Review by FluffyGhoster — APPROVED** [review]
All good then

File: code/modules/clothing/under/accessories/armor.dm (hunk 2 above)
Comment (Clem-The-Duck) [reply code/modules/clothing/under/accessories/armor.dm]: Made the SCC heavy set have slowdown for the sake of consistency incase the armour set is made available in future.

**Comment by Clem-The-Duck** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]
Yeah why not.
